code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -331,3 +331,85 @@
 	icon_state = "closed_mcart_y"
 	icon_opened = "open_mcart_y"
 	icon_closed = "closed_mcart_y"
+
+/obj/structure/closet/crate/mass_produced_crate
+	name = "Mass Produced Crate"
+	desc = "A rectangular steel crate. Cannot be welded for metal."
+	icon = 'icons/obj/structures/crates.dmi'
+	icon_state = "closed_basic"
+	icon_opened = "open_basic"
+	icon_closed = "closed_basic"
+	max_integrity = 5 //hopefully makes it so req crate spam can easily be destroyed
+	materials = list(/datum/material/metal = 200)
+	drop_material = FALSE
```

Comment:
Minor thing but make this null, not FALSE.

Reply:
done and done